internal/environment/environment.go

```diff
@@ -92,7 +92,7 @@ func GetLanguages() string {
 }
 
 func CreateGitRelease() bool {
-	return os.Getenv("INPUT_CREATE_RELEASE") == "true" || IsLanguagePublished("php") || IsLanguagePublished("terraform")
+	return os.Getenv("INPUT_CREATE_RELEASE") == "true" || IsLanguagePublished("php") || IsLanguagePublished("terraform") || IsLanguagePublished("swift")
```

Comment:
Does swift always need a github release associated with any published packages? Generally this check is here just for those languages whose package manager doesn't work without creating a github release

Reply:
sorry just seen further down you are just looking to do github releases. No package manager for swift?

Reply:
there's cocoapods which is a separate dependency manager: for now we're just supporting the [Swift Package Manager](https://www.swift.org/package-manager/) which only needs a git remote + tag to resolve, so GH release + tag is perfect for this.